src/Markdig/Helpers/CharHelper.cs

```diff
@@ -81,6 +86,26 @@ public static bool IsRomanLetterUpperPartial(char c)
         }
 
         [MethodImpl(MethodImplOptionPortable.AggressiveInlining)]
+        public static int RomanToArabic(string text)
```

Comment:
Would this be better if we just converted the entire string `ToUpper` at the start?